examples/lessons/08/app.js

```diff
@@ -111,47 +110,53 @@ const animationLoop = new AnimationLoop({
       .rotateXYZ([tick * 0.01, tick * 0.01, tick * 0.01])
       .multiplyRight(cube.matrix);
 
+
+    const {
+      blending,
+      lighting,
+      ambient,
+      direction
+    } = getControls();
+
+    if (blending.checked) {
+      gl.blendFunc(gl.SRC_ALPHA, gl.ONE);
```

Comment:
Instead of multiple `gl.func` you can use `setParameters` :
```
    setParameters(gl, {
      blend: true
      blendFunc: [gl.SRC_ALPHA, gl.ONE],
      depthTest: false
    });
```
similarly in other places too.